Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -248,7 +248,7 @@ void VirtualFileSystem::sync_filesystems()
     });
 
     for (auto& fs : file_systems)
-        fs->flush_writes();
+        MUST(fs->flush_writes());
```

Comment:
Same comment here about the panic. Especially since I think (?) userspace can call a syscall to flush all filesystems (sys$sync?)

Reply:
I appreciate the feedback, I'll make sure to include the changes you recommended.